docusaurus/docs/cookie.md

```diff
@@ -0,0 +1,61 @@
+---
+id: cookie
+title: '@idearium/cookie'
+---
+
+Idearium cookie wrapper.
+
+## Installation
+
+```shell
+$ yarn add -E @idearium/cookie
+```
+
+### Beta installation
+
+If you need to install a beta version, you can:
+
+```shell
+$ yarn add -E @idearium/cookie@beta
+```
+
+## Usage
+
+To use `@idearium/cookie`, simply require it at the top of your file.
+
+`@idearium/cookie` exports a number of pure functions to help you work with cookies:
+
+-   `create` - creates a cookie.
+-   `find` - finds a cookie in a HTTP cookie header string.
+
+### create
+
+To create a new cookie:
+
+```js
+const { create } = require('@idearium/cookie');
+
+await create({
+    cookie: 'cookie string value',
+    name: 'foo',
+});
+```
+
+The create function uses the [cookie](https://www.npmjs.com/package/cookie#options-1) library in the background and can accept any `serialize` options listed there.
```

Comment:
Below this, could you please do a little more complex example and set `httpOnly` and `secure` perhaps?

Reply:
Also, I think a more complete example in some middleware using `res.set('set-cookie', cookieStr);` might be good.